src/components/common/ConnectWallet/WalletDetails.tsx

```diff
@@ -6,7 +6,12 @@ import type { ConnectedWallet } from '@/services/onboard'
 import useConnectWallet from '@/components/common/ConnectWallet/useConnectWallet'
 
 const WalletDetails = ({ onConnect }: { onConnect?: (wallet?: ConnectedWallet) => void }): ReactElement => {
```

Comment:
Checking the places we use `WalletDetails` it looks like `onConnect` doesn't have any params anymore so it could just be `onConnect?: () => void` now.